packages/ember-glimmer/lib/syntax/experimental-component-manager.js

```diff
@@ -0,0 +1,308 @@
+import { OWNER } from 'ember-utils';
```

Comment:
Not sure about this file naming.  I'd prefer `packages/ember-glimmer/lib/component-managers/curly.js`.